samples/client/petstore/typescript-fetch/builds/default/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-4.1.1-SNAPSHOT
\ No newline at end of file
+unset
```

Comment:
I'm not sure why the version is clobbered when I run the sample code re-generation. I can manually undo if necessary.

Reply:
are you sure you ran `mvn clean package` before generating the samples? are you based on the latest master?

Reply:
I'm based on 024f46814cd7071841c08a4363fa73c7d26a2aac (master circa 2019-08-18).

If I run
```
mvn clean package
./bin/typescript-fetch-petstore-all.sh
```

on my current branch (where the `.openapi-generator/VERSION` files already have "unset"), no sample files are changed. If I run it on master, the `.openapi-generator/VERSION` files are overwritten to "unset".

FWIW, I (but no one else on my team) had this problem when running a brew-installed jar of of the old swagger-codegen, which is part of why we switched to running it in Docker... there could just be something weird in my local env.

Reply:
fixed manually